stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/notifications/internal/VideoPushDelegate.kt

```diff
@@ -102,7 +104,7 @@ internal class VideoPushDelegate(
                     context = context,
                     user = user,
                     token = userToken,
-                    apiKey = userDataStore.apiKey.value,
+                    apiKey = userDataStore.apiKey.first(),
```

Comment:
This is never null and the function doesn't take null. 

```
    public val apiKey: Flow<ApiKey> =
        data.map { it?.apiKey.orEmpty() }
```
It creates an empty API key - I haven't changed this logic. That's why I have to use `first()` here so that it compiles and expects a non-null value.